src/docs/sphinx/dev_guide/profiling.rst

```diff
@@ -109,3 +109,21 @@ of this file, use `cali-query <https://software.llnl.gov/Caliper/tools.html#cali
 
 To view this data with SPOT, open a browser, navigate to the SPOT server (e.g. `LC <https://lc.llnl.gov/spot2>`_), and open the directory containing one or more ``.cali`` files.  For more information, watch this recorded `tutorial <https://www.youtube.com/watch?v=p8gjA6rbpvo>`_.
 
+Benchmarking Serac
+------------------
+
+To run all of Serac's benchmarks in one command, first make sure Serac is configured
+with benchmarking enabled (off by default). Then, run the build target ``run_benchmarks``.
+
+.. code-block:: bash
+
+  ./config-build.py -hc <host config file> -DENABLE_BENCHMARKS=ON
+  cd <serac build location>
+  make -j
+  make run_benchmarks
+  find . -name "*.cali" -print0 | xargs -0 mv -t .
```

Comment:
@white238 Added this part to the docs just now. The Caliper files will not be located at the root of the build dir by default.

It's somewhat unfortunate we have to do that command but it might be avoidable in the future if `blt_add_test` and `blt_add_benchmark` added a `WORKING_DIRECTORY` option. https://cmake.org/cmake/help/latest/command/add_test.html#add-test